packages/eas-cli/src/commands/build/create.ts

```diff
@@ -55,14 +61,19 @@ export default class BuildCreate extends Command {
       return;
     }
 
+    await ensureProjectConfiguredAsync(projectDir);
```

Comment:
Does it make sense to call this function when the platform is still not resolved?

Reply:
it was intentional, explained under the comment bellow